### DISCUSSION TITLE
IPV6 Passthrough

### DISCUSSION THREAD

**Discussion by montroseblue** [iamromulan/cellular-modem-wiki#24] [Help - Quectel]
I'm not able to get IPV6 Passthrough working on the MCUZone board + RM520N-GL + TMobile Business Tablet SIM + Archer A7 Router. I tried IPV4V6 and IPV6 on APN. AT+QMAP="WWAN" does yield an IPV6 address as well as IPV4, but it doesn't passthrough. Only IPV4 is working. One issue I'm having is usually gaming won't work. I'm not able to bring up any servers. I have to use VPN for it to work.

Anyone know how to get it to IPV6?

**Comment by vbaskoff** [comment]
Same using 5G2PHY + RM520N-GL

Seems that IPv6 pass-through is a bit tricky due to nature mobile ISP assigning actual IPv6 to modem and than rely on client behavior (SLAAC, 464XLAT(CLAT) etc.).

I am trying to get my IPv6 /64 prefix form Orange Poland (over 4G/5G connection) which seems assigned using SLAAC and fully rely on CLAT support on the client side.

I have tried to use DHCPv6 client at my Mikrotik behind 5G2PHY (in pass-through) to request /64 prefix from Orange - no luck so far.

**Comment by ghost** [comment]
Anybody reading this: AVOID Macuzone boards. They truly suck. I had their V3 PoE board and their V2 Non PoE boards. Both the boards had difficulty in maintaining a steady connection with my Ubiquiti Cloud Gateway Fiber. IPv6 never worked on them as well.

I then switched to iamromulan specified PHY board and it worked flawlessly. The only thing is it needed a 12V active to passive PoE converter adapter. Ubiquiti sells such adapters on their store page under camera accessories section. 

The Macuzone V3 had active PoE so it could be powered directly by UCG-Fiber's Port no. 4. But apart from that it had nothing going for it. If there are constant 5 min disconnections irrespective of powering method, I sold it off to my friend who is fine feeding it directly to his PC. No disconnection issue for him.

**Reply by carp4** [reply]
Agree. I a MCUZone one and it was horrible too. Switched to the Chester Tech Repairs Pocket Rocket and DualQ 5G2PHY sled. Both work awesome.

**Reply by ghost** [reply]
> Agree. I a MCUZone one and it was horrible too. Switched to the Chester Tech Repairs Pocket Rocket and DualQ 5G2PHY sled. Both work awesome.

Do you have a linkbfor DualQ 5G2PHY sled?

**Reply by carp4** [reply]
From iamromulan:

https://discord.gg/vGjJEMTDb5

From Chester Tech Repair:

https://chestertechrepairs.com/

(But at the time of this post it says out of stock. He has the v1 and v2 pocket rocket though and they are great.)

From Lei Ming:

https://www.rework.network/collections/lte-home-gateway/products/5g2phy
